build/cargo.gni

```diff
@@ -11,38 +11,26 @@ if (is_mac) {
 }
 
 declare_args() {
-  rustup_path = rebase_path("//brave/build/rustup/", root_build_dir)
+  rustup_path = rebase_path("//brave/build/rustup")
 }
 
 template("rust_ffi") {
   config_target_name = target_name + "_external_config"
   cpp_target_name = "rust_ffi"
 
-  if (is_mac) {
-    assert(
-        defined(invoker.shared_library_define),
-        "Must specify a shared library define for importing symbols on mac ex: ADBLOCK_SHARED_LIB")
-    assert(
-        defined(invoker.implementation_define),
-        "Must specify an implementation define for exporting symbols on mac ex: ADBLOCK_IMPLEMENTATION")
-  } else {
-    not_needed(invoker,
-               [
-                 "shared_library_define",
-                 "implementation_define",
-               ])
-  }
```

Comment:
Removing this breaks builds on non-Mac

Reply:
cc; @bridiver - I know he'll be looking at bringing the cxx functionality in with another PR

Reply:
I think this is fixed now? I don't remember getting the error anymore